app/views/downloads/guis/index.html.haml

```diff
@@ -28,109 +28,127 @@
         =link_to(image_tag('guis/git-cola@2x.png', {:width => '294', :height => '166'}), "https://git-cola.github.io/")
         %h4= link_to "git-cola", "https://git-cola.github.io/"
         %p.description
           <strong>Platforms:</strong> Windows, Mac, Linux</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> GNU GPL
       %li.mac
         =link_to(image_tag('guis/gitbox@2x.png', {:width => '294', :height => '166'}), "http://www.gitboxapp.com/")
         %h4= link_to "Gitbox", "http://www.gitboxapp.com/"
         %p.description
           <strong>Platforms:</strong> Mac</br>
-          <strong>Price:</strong> $14.99
+          <strong>Price:</strong> $14.99</br>
+          <strong>License:</strong> Proprietary
       %li.mac.windows.linux
         =link_to(image_tag('guis/cycligent@2x.png', {:width => '294', :height => '166'}), "https://www.cycligent.com/git-tool")
         %h4= link_to "Cycligent Git Tool", "https://www.cycligent.com/git-tool"
         %p.description
           <strong>Platforms:</strong> Windows, Mac, Linux</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> Proprietary
       %li.mac.windows.linux
         =link_to(image_tag('guis/giteye@2x.png', {:width => '294', :height => '166'}), "http://www.giteyeapp.com/")
         %h4= link_to "GitEye", "http://www.giteyeapp.com/"
         %p.description
           <strong>Platforms:</strong> Windows, Mac, Linux</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> Proprietary
       %li.mac
         =link_to(image_tag('guis/fork@2x.png', {:width => '294', :height => '166'}), "https://git-fork.com/")
         %h4= link_to "Fork", "https://git-fork.com/"
         %p.description
           <strong>Platforms:</strong> Mac</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> Proprietary
       %li.mac.windows.linux
         =link_to(image_tag('guis/gitahead@2x.png', {:width => '294', :height => '166'}), "https://www.gitahead.com/")
         %h4= link_to "GitAhead", "https://www.gitahead.com/"
         %p.description
           <strong>Platforms:</strong> Windows, Mac, Linux</br>
-          <strong>Price:</strong> $29/user (Free 15 day trial)
+          <strong>Price:</strong> $29/user (Free 15 day trial)</br>
+          <strong>License:</strong> Proprietary
       %li.linux
         =link_to(image_tag('guis/giggle@2x.png', {:width => '294', :height => '166'}), "https://wiki.gnome.org/Apps/giggle/")
         %h4= link_to "giggle", "https://wiki.gnome.org/Apps/giggle/"
         %p.description
           <strong>Platforms:</strong> Linux</br>
-          <strong>Price:</strong> Free
+          <strong>Price:</strong> Free</br>
+          <strong>License:</strong> GNU GPL
 
```

Comment:
two more other GUIs were added in https://github.com/git/git-scm.com/pull/1029 and https://github.com/git/git-scm.com/pull/782, can you rebase to solve the conflicts and add the information for those new GUIs please?

Reply:
useful info:

* https://github.com/FredrikNoren/ungit#license-mit
* https://github.com/FabriceSalvaire/CodeReview/blob/master/GPL-V3.0.txt